src/test/java/plugins/CompressArtifactsPluginTest.java

```diff
@@ -101,11 +119,6 @@ public void archiveLargerThan4GInTotal() throws Exception {
                 job.name
         ));
         assertThat(length, greaterThanOrEqualTo(4L * 1024 * 1024 * 1024));
-
-        build.getArtifact("stuff.jar").open();
-        for (int i = 1; i <= 70; i++) {
-            build.getArtifact("stuff." + i + "42.jar").open();
```

Comment:
would attempt to open `stuff.7042.jar` which is not created and would not exist yet the test would still pass anyway.